docroot/profiles/custom/cgov_site/modules/custom/cgov_saml_auth_config/config/install/cgov_saml_auth_config.settings.yml

```diff
@@ -0,0 +1,37 @@
+langcode: en
+default_langcode: en
+activate: false
+mail_attr: https://federation.nih.gov/person/Mail
+unique_id: https://federation.nih.gov/person/SamAccountName
+user_name: https://federation.nih.gov/person/SamAccountName
+auth_source: cancergov-sp
+login_link_display_name: 'NIH Login'
+header_no_cache: true
+role:
+  population: null
+  eval_every_time: false
+register_users: true
```

Comment:
I would like it if we have to add users on our own. With tens of thousands of NIH users, I would like to avoid people hitting our servers and automatically getting accounts.